api/services/ConfigService.js

```diff
@@ -0,0 +1,176 @@
+/* globals Config */
+
+const Promise = require('bluebird');
+
+/**
+ * nanocloudConfigValue returns the value associated to the config variable's
+ * name. It will retreive the value from the environment variable if any and
+ * ensure that the type of the varriable is the same as the defaultValue.
+ * If default value is an object, the environment variable is expected to be a
+ * serialized JSON object.
+ * It it's an array, it's expected to be a serialized JSON array.
+ * Otherwise, it returns the defaultValue.
+ *
+ * @method nanocloudConfigValue
+ * @private
+ * @param {String} name Name of the config variable
+ * @param {String} defaultValue The value to return if the environment variable
+ * "name" is not set.
+ * @return {Object} The environment variable if found (casted to the type of
+ * defaultValue). defaultValue otherwise.
+ */
+function nanocloudConfigValue(name, defaultValue) {
+  let value;
+
+  if (process.env.hasOwnProperty(name)) {
+    let type;
+
+    let value = process.env[name];
+
+    if (Array.isArray(defaultValue)) {
+      type = 'array';
+    } else {
+      type = (typeof defaultValue);
+    }
+
+    switch (type) {
+      case 'number':
+        value = parseInt(value, 10);
+        if (Number.isNaN(value)) {
+          throw new Error(`Config variable '${name}' must be an number.`);
+        }
+        break;
+
+      case 'array':
+        value = JSON.parse(value);
+        if (!Array.isArray(value)) {
+          throw new Error(`Config variable '${name}' must be an array.`);
+        }
+        break;
+
+      case 'object':
+        value = JSON.parse(value);
+        if (typeof value !== 'object') {
+          throw new Error(`Config variable '${name}' must be an object.`);
+        }
+        break;
+
+      case 'boolean':
+        if (value === 'true') {
+          value = true;
+        } else if (value === 'false') {
+          value = false;
+        }
+        throw new Error(`Config variable '${name}' must be an boolean.`);
+
+   // case 'string':
```

Comment:
No. It's there to explicit the 'string' case (which doesn't need special treatment).